src/Fixer/Phpdoc/PhpdocAnnotationWithoutDotFixer.php

```diff
@@ -90,9 +90,14 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 $optionalTypeRegEx = $annotation->supportTypes()
                     ? sprintf('(?:%s\s+(?:\$\w+\s+)?)?', preg_quote(implode('|', $annotation->getTypes())))
                     : '';
-                $content = preg_replace_callback('/^(\s*\*\s*@\w+\s+'.$optionalTypeRegEx.')(.*)$/', function (array $matches) {
-                    return $matches[1].lcfirst($matches[2]);
-                }, $startLine->getContent(), 1);
+                $content = preg_replace_callback(
+                    '/^(\s*\*\s*@\w+\s+'.$optionalTypeRegEx.')(\p{Lu}?(?=\p{Ll}|\p{Zs}))(.*)$/',
+                    function (array $matches) {
+                        return $matches[1].strtolower($matches[2]).$matches[3];
```

Comment:
That makes sense, although, to do it right, we would likely want to detect the encoding of the file. Grepping the source, it doesn't appear this is done outside of the cache. It would probably make the most sense for this to be done once per file outside of any particular fixer.